controller/src/main/java/org/jboss/as/controller/ListAttributeDefinition.java

```diff
@@ -61,6 +62,15 @@ public ParameterValidator getElementValidator() {
         return elementValidator;
     }
 
+    /**
+     * Returns an AttributeDefinition describing the content of the list.
+     * @return an AttributeDefinition describing the content of the list.
+     * @throws an UnsupportedOperationException if the expected AttributeDefinition of list values is not defined.
```

Comment:
I don't think that javadoc is accurate anymore. Instead it should mention that it returns `null` if the list does not define an attribute definition for its item.